### ISSUE TITLE
Add a 'required time' indicator to the drop item menu

### ISSUE THREAD

**Issue by MelacholicMimic** [CleverRaven/Cataclysm-DDA#57121]
### Is your feature request related to a problem? Please describe.

Dropping my backpack before combat is one of the most repeated actions I do in the game, yet there is no indicator anywhere how long it actually takes me to do. Will I get hit if I drop it now? Should I just accept the to-hit penalty? Should I use some stamina to run away and then drop it? I picked up 10 items since last time, how much longer does it take now? Those are all tactical decisions I have no accurate way to decide on. 

### Solution you would like.

Something similar to this: 
 ```
ITEMS TO DROP   
    Backpack: 517 moves 
    Something else: 127 moves
Total : 644 moves
```
I would expect the moves here to be the final value, as it is the relevant information. E.g. It does really take 644 moves, not 1288 because my speed is 50. 

### Describe alternatives you have considered.

Provide only the total cost for dropping in the menu header.

**Comment by PLUUZ** [comment]
I would also really like to see this, to complement the way the wield costs are already shown. With the addition of nested containers/ pockets, dropping things should become really important part of combat as well, but it's somewhat hampered because 

1. the game doesn't show how long it takes, and 
3. It often takes a really unrealistically long amount of time to drop some items like backpacks, swag bags, etc.  

The fact that both 1 and 2 exist results in many situations where you intuitively think you can drop a backpack or whatever before a zombie closes on you, but it takes like 6 seconds and he's in your face by the time you finish dropping. 

I recently made an issue about this for the swag bag #56986 which also takes like 4-6 seconds to take off. From my brief, uneducated foray into this repo it seems like there's a flat 250 move cost to takeoff everything, plus costs based on volume or weight or encumbrance I guess. In that issue I linked some stale PR where someone was toying with adding custom takeoff movement costs to items.

Basically, both the two things  could be addressed -  the game signaling the drop/takeoff movement costs something like how it signals wield movement costs, and discussion of the many items like backpacks, swag bags etc that seem to be too slow to take off.
